misc/stdlib_diff/README.md

```diff
@@ -1,6 +1,8 @@
 # stdlibs_diff
 
-stdlibs_diff is a tool that generates an html report indicating differences between gno standard libraries and go standrad libraries
+stdlibs_diff is a tool that generates an html report indicating differences between gno standard libraries and go standard libraries. 
+
+⚠️ When using goenv to set the src or the destination, the Go version will be the one from the toolchain on go.mod.
```

Comment:
I'm not sure this comment makes sense, especially as a warning. Isn't it about the concept of goenv?

Reply:
Hello thanks for your review, maybe It will do more sense as a comment on the gh-pages.yml? 

The idea was to mention what Golang version is used for the stdlibs comparaison. This was a need from the issue